plugins/org.jkiss.dbeaver.model.lsm/src/org/jkiss/dbeaver/model/lsm/LSMAnalyzerImpl.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.lsm;
+
+import org.antlr.v4.runtime.ANTLRErrorListener;
+import org.antlr.v4.runtime.ConsoleErrorListener;
+import org.antlr.v4.runtime.Lexer;
+import org.antlr.v4.runtime.RecognitionException;
+import org.antlr.v4.runtime.atn.PredictionMode;
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.model.lsm.mapping.SyntaxModel;
+import org.jkiss.dbeaver.model.lsm.mapping.SyntaxModelMappingResult;
+import org.jkiss.dbeaver.model.lsm.sql.impl.SelectStatement;
+import org.jkiss.dbeaver.model.stm.STMParserOverrides;
+import org.jkiss.dbeaver.model.stm.STMSource;
+import org.jkiss.dbeaver.model.stm.STMTreeRuleNode;
+import org.jkiss.utils.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class LSMAnalyzerImpl<TLexer extends Lexer, TParser extends STMParserOverrides> implements LSMAnalyzer {
+    private static final Logger log = LoggerFactory.getLogger(LSMAnalyzerImpl.class);
```

Comment:
Please our Log instead of slf4j

Reply:
It leads to the cyclic dependency between org.jkiss.dbeaver.model and org.jkiss.dbeaver.model.lsm